src/app/src/main/kotlin/org/vaccineimpact/orderlyweb/db/repositories/WorkflowRunRepository.kt

```diff
@@ -0,0 +1,31 @@
+package org.vaccineimpact.orderlyweb.db.repositories
+
+import com.google.gson.Gson
+import org.vaccineimpact.orderlyweb.db.JooqContext
+import org.vaccineimpact.orderlyweb.db.Tables
+import org.vaccineimpact.orderlyweb.models.WorkflowRun
+import java.sql.Timestamp
+
+interface WorkflowRunRepository
+{
+    fun addWorkflowRun(workflowRun: WorkflowRun)
+}
+
+class OrderlyWebWorkflowRunRepository : WorkflowRunRepository
+{
+    override fun addWorkflowRun(workflowRun: WorkflowRun)
+    {
+        JooqContext().use {
+            it.dsl.insertInto(Tables.ORDERLYWEB_WORKFLOW_RUN)
+                .set(Tables.ORDERLYWEB_WORKFLOW_RUN.NAME, workflowRun.name)
+                .set(Tables.ORDERLYWEB_WORKFLOW_RUN.KEY, workflowRun.key)
+                .set(Tables.ORDERLYWEB_WORKFLOW_RUN.EMAIL, workflowRun.user)
```

Comment:
is there a reason why user can't be "email" in the model, just for uniformity?

Reply:
In the database it's `EMAIL` for consistency with other tables, and so that foreign key columns have the same name. In the model it's for `user` consistency with other models, and also because a workflow doesn't have an email address as such but it does have a user. So I think it's inconsistent in the least possible way.

Reply:
This class `org.vaccineimpact.orderlyweb.models.UserDetails` has it's own repository with user info, which means `user` shouldn't qualify as `email`. For clarity, I still think you should replace user with email.

Reply:
Done